projects/js-packages/publicize-components/src/components/admin-page/toggles/social-image-generator-toggle/index.tsx

```diff
@@ -60,7 +60,7 @@ const SocialImageGeneratorToggle: React.FC< SocialImageGeneratorToggleProps > =
 				disabled={ isUpdating || ! isEnabled }
 				onClick={ open }
 			>
-				{ __( 'Change default Template and Image', 'jetpack-publicize-components' ) }
+				{ __( 'Change defaults', 'jetpack-publicize-components' ) }
```

Comment:
Completely optional suggestion:

What do you think about "Configure defaults"?